nixos/modules/hardware/video/backlight/mba6x_bl.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, ... }:
+
+with lib;
+
+let cfg = config.services.mba6x_bl;
+    kernel = config.boot.kernelPackages;
+    modprobe = "${config.system.sbin.modprobe}/sbin/modprobe";
+in
+{
+  options = {
+    services.mba6x_bl = {
+      enable = mkEnableOption "the mba6x_bl backlight driver for MacBook Air 6,1 and 6,2";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    boot.extraModulePackages = [ kernel.mba6x_bl ];
+    systemd.services.mba6x_blWorkaround = {
+      description = "Remove and reload mba6x to workaround no brightness bug (#43)";
```

Comment:
It's not clear to which project Issue 43 belongs. Maybe you can add a hint where to find it.